RxBluetoothKit.podspec

```diff
@@ -20,5 +20,5 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Source/*.swift'
   s.frameworks   = 'CoreBluetooth'
-  s.dependency 'RxSwift', '4.0.0-rc.0'
+  s.dependency 'RxSwift', '4.0.0'
```

Comment:
It seems like you'd much rather have a dependency like `s.dependency 'RxSwift', '~> 4.0'`, instead of a hard dependency on `4.0.0`.

Reply:
That's right, thank you!